profiles/schema.py

```diff
@@ -0,0 +1,118 @@
+import graphene
+from django.conf import settings
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required
+from munigeo.models import AdministrativeDivision
+from thesaurus.models import Concept
+
+from .models import Profile
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+        fields = ("code",)
+
+    vocabulary = graphene.String()
+    label = graphene.String()
+
+    def resolve_vocabulary(self, info, **kwargs):
+        return self.vocabulary.prefix
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+        fields = ("type", "children", "origin_id", "ocd_id", "municipality")
+
+    type = graphene.String()
+    name = graphene.String()
+
+
+class ProfileType(DjangoObjectType):
+    class Meta:
+        model = Profile
+        fields = (
+            "nickname",
+            "image",
+            "email",
+            "phone",
+            "language",
+            "contact_method",
+            "concepts_of_interest",
+            "divisions_of_interest",
+        )
+
+    concepts_of_interest = graphene.List(ConceptType)
+    divisions_of_interest = graphene.List(AdministrativeDivisionType)
+
+    def resolve_concepts_of_interest(self, info, **kwargs):
+        return self.concepts_of_interest.all()
+
+    def resolve_divisions_of_interest(self, info, **kwargs):
+        return self.divisions_of_interest.all()
+
+
+Language = graphene.Enum("Language", [(l[1].upper(), l[0]) for l in settings.LANGUAGES])
+ContactMethod = graphene.Enum(
+    "ContactMethod", [(cm[1].upper(), cm[0]) for cm in settings.CONTACT_METHODS]
+)
+
+
+class ProfileInput(graphene.InputObjectType):
+    nickname = graphene.String()
+    image = graphene.String()
+    email = graphene.String()
+    phone = graphene.String()
+    language = Language()
+    contact_method = ContactMethod()
+    concepts_of_interest = graphene.List(graphene.String)
+    divisions_of_interest = graphene.List(graphene.String)
+
+
+class CreateProfile(graphene.Mutation):
+    class Arguments:
+        profile = ProfileInput(required=True)
+
+    profile = graphene.Field(ProfileType)
+
+    @login_required
+    def mutate(self, info, **kwargs):
+        profile_data = kwargs.pop("profile")
+        concepts_of_interest = profile_data.pop("concepts_of_interest", [])
+        divisions_of_interest = profile_data.pop("divisions_of_interest", [])
+        profile_data["user"] = info.context.user
+        profile = Profile.objects.create(**profile_data)
```

Comment:
creating profile is only half of the logic, we also need to make it possible to update the profile 🤔 
maybe rename the mutation to `UpdateProfile` and then do `get_or_create`